.changeset/famous-pants-agree.md

```diff
@@ -0,0 +1,6 @@
+---
+'@builder.io/sdk': patch
+'@builder.io/react': patch
+---
+
+Fix: Content API invocations will have `includeRefs` set to `true` by default and will not have `enrich` as part of its query-params
```

Comment:
I would prefer not to mention `enrich` at all in here as it might confuse customers:

```suggestion
Fix: Content API invocations will have `includeRefs` set to `true` by default.
```